.github/workflows/test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 10.x, 12.x, 14.x, 16.x, 17.x, 18.x, 19.x]
```

Comment:
Removing all node versions except 19.x significantly reduces coverage and does not make sense.  PLan is to have test coverage across node versions at least mirror exceljs